ServerCore/Pages/Teams/Create.cshtml.cs

```diff
@@ -16,7 +16,12 @@ public CreateModel(PuzzleServerContext context)
         }
 
         public IActionResult OnGet()
-        {
+        {        
+            if (!Event.IsTeamRegistrationActive)
```

Comment:
It would be good to check this in OnPostAsync too to avoid issues like a team loading the page before the deadline and submitting afterward

Reply:
Good point. I'll just need to check it in Create's OnPostAsync, right? Not anywhere else?

Reply:
I think that covers it